WolvenKit.Witcher3Formats/W3Speech/SpeechManager.cs

```diff
@@ -54,7 +54,15 @@ public override void LoadAll(string exedir)
                     .SelectMany(dir => Directory.GetFiles(dir ?? "", "*.w3speech", SearchOption.AllDirectories)
                         .OrderBy(k => k)))
                 {
-                    LoadArchive(file);
+                    try
+                    {
+                        LoadArchive(file);
+                    }
+                    catch (Exception e)
+                    {
```

Comment:
@Traderain we are getting constant stream length exceptions here for the DLC. needs investigation